src/cr-logging/CrLogstashJsonFormatter.cs

```diff
@@ -0,0 +1,83 @@
+﻿// <copyright file="CrLogstashJsonFormatter.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
+
+namespace CR.Logging
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Serilog.Events;
+    using Serilog.Formatting;
+    using Serilog.Formatting.Json;
+
+    /// <summary>
+    /// Formatter to output into json which is readable by Logstash
+    /// </summary>
+    public class CrLogstashJsonFormatter : ITextFormatter
+    {
+        private static readonly JsonValueFormatter ValueFormatter = new JsonValueFormatter();
+
+        /// <summary>
+        /// Formats a log event.
```

Comment:
Into what format does it turn the log event, could do with more detail explain what we are doing.